src/main/java/codeu/controller/ChatServlet.java

```diff
@@ -104,19 +104,28 @@ void setEventStore(EventStore eventStore) {
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response)
       throws IOException, ServletException {
+
     String requestUrl = request.getRequestURI();
-    String conversationTitle = requestUrl.substring("/chat/".length());
+    String conversationIdAsString = requestUrl.substring("/chat/".length());
+    UUID conversationId = null;
 
-    Conversation conversation = conversationStore.getConversationWithTitle(conversationTitle);
+    try{
```

Comment:
This seems to be duplicated below. Have a method to extract the id from the string?